misc/dbt-materialize/MAINTAINERS.md

```diff
@@ -88,11 +81,10 @@ pytest -s
 ```
 
 Don't drop the schema in `materialized` after a test failure, so that you can
```

Comment:
Is it fine to just default to `mzcompose` instead of trying to provide instructions to run tests ad-hoc using `pytest`? IIUC, this now would entail adapting `conftest-py` to connect to a cloud instance, and the custom materialization tests would require at least Redpanda (and Postgres) to be running somewhere.

Reply:
I think so! I do run pytest locally just for speed, but end up needing to run mzcompose anyway for full integration tests. But I would ask that we include `-s` with pytest to see the logs of the test run. 

I also wonder if we could rig up a docker volume w/ the mzcompose set up so we can see the compiled SQL getting run for these test conditions. (Another reason I use pytest locally is to actually explore the relations getting created in Materialize).